api/src/main/java/org/openmrs/module/ipd/api/model/MedicationAdministrationNote.java

```diff
@@ -55,6 +55,38 @@ public class MedicationAdministrationNote extends BaseOpenmrsData {
 	@Column(name = "text", length=65535)
 	private String text;
 
+	/**
+	 * Amendment reason - why this note was amended
+	 */
+	@Column(name = "amended_reason", nullable = true)
```

Comment:
@SooryaKumaranC-tw we need a new field for amended text. Instead of overriding the old notes field, we need to have store under new column